ops/charm.py

```diff
@@ -714,24 +716,24 @@ def restore(self, snapshot: Dict[str, Any]):
 
 
 class SecretChangedEvent(SecretEvent):
-    """Event raised by Juju on the observer when the secret owner changes its contents.
+    """Event triggered by Juju on the observer when the secret owner changes its contents.
```

Comment:
'observer' is a loaded term in this context. 
besides, none of the other events have the 'by juju'  part.

how about
"
Event emitted on the charm when the contents of a secret it is tracking are changed by its owner..
"

Reply:
Fair call. However, we do use the term "secret observer" vs "secret owner". I've clarified by spelling it out, and removed the "by Juju" phrases.